SpatialGDK/Documentation/content/schema.md

```diff
@@ -0,0 +1,39 @@
+# Schema
+
+SpatialOS uses [schema (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/concepts/schema#schema) to generate APIs specific to the components in your project. These APIs define how you can operate on entity components within SpatialOS.
+
+## Generating schema
+
+To generate schema, select the **Schema** button in the [GDK Toolbar]({{urlRoot}}/content/toolbars#buttons). The GDK automatically iterates through classes with replicated properties to generate the required schema files and then updates the [SchemaDatabase]({{urlRoot}}/content/glossary#schemadatabase).
+
+![Toolbar]({{assetRoot}}assets/screen-grabs/toolbar/schema-button.png)
+
+_Image: In the GDK toolbar in the Unreal Editor, select **Schema**_
+
+As the GDK automatically generates all the schema you need, you do not have to write or edit schema manually when using the GDK.
+
+### When to generate schema
+
+You must generate schema when you add or change any [replicated properties (Unreal documentation)](https://docs.unrealengine.com/en-US/Gameplay/Networking/Actors/Properties) that you want to deploy to SpatialOS.
+
+The GDK only generates schema for classes currently loaded into memory. This means if your project uses [sublevels](<https://docs.unrealengine.com/en-us/Engine/Levels/LevelsWindow>), you’ll need to load them in addition to your map, before generating schema.
+
+## Deleting schema
+
+Whenever you generate schema, the GDK checks the [SchemaDatabase]({{urlRoot}}/content/glossary#schemadatabase) and all of your project classes currently loaded into memory, and removes any classes referenced in the `SchemaDatabase` that no longer exist.
```

Comment:
This sentence and the next kinda say the same thing. Essentially - "When generating schema, the GDK verifies that anything previously saved in the SchemaDatabase still exists. If the class has been deleted, it's also removed from the SchemaDatabase."

Reply:
>New text
When you generate schema, the GDK verifies that any classes referenced in the [SchemaDatabase]({{urlRoot}}/content/glossary#schemadatabase) still exist. If you delete a class the GDK removes it from the SchemaDatabase the next time you generate schema.